src/OpenSage.Game/Gui/Wnd/Controls/ListBox.cs

```diff
@@ -289,17 +300,36 @@ public int MaxDisplay
             }
         }
 
-        public void UpdateSelectedItem()
+        private void SetSelectedItem()
         {
             for (var i = 0; i < _items.Length; i++)
             {
                 var item = (ListBoxItem) Controls[i];
 
                 if (i == _selectedIndex)
                 {
-                    item.BackgroundImage = IsMouseOver
-                        ? SelectedItemHoverBackgroundImage
-                        : SelectedItemBackgroundImage;
+                    item.BackgroundImage = SelectedItemBackgroundImage;
+                }
+                else
+                {
+                    item.BackgroundImage = null;
+                }
+            }
+        }
+
+        private void SetHoveredItem()
+        {
+            for (var i = 0; i < _items.Length; i++)
```

Comment:
If we want to get fancy and micro-optimize everything, I think this (and `SetSelectedItem`) could be implemented without a loop. Instead of using `_hoveredIndex` pass the previous and new hovered index as arguments, and then just do something like this:

```csharp

if (previousIndex == nextIndex) return;

var previousItem = Controls[previousIndex] as ListBoxItem;
previousItem.BackgroundImage = previousIndex === _selectedIndex ? SelectedItemBackgroundImage : null;

var nextItem = Controls[nextIndex] as ListBoxItem;
nextItem.BackgroundImage = nextIndex === _selectedIndex ? SelectedItemHoverBackgroundImage : null;
```

(Untested, feel free to use full `if`s if you think it makes it more readable.)